modules/cnv-deleting-hco-subscription.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="cnv-deleting-hco-subscription_{context}"]
+= Deleting the KubeVirt HyperConverged Cluster Operator catalog subscription
+
+To finish uninstalling {ProductName}, delete your KubeVirt HyperConverged
+Cluster Operator catalog subscription.
+
+.Prerequisites
+
+* An active *KubeVirt HyperConverged Cluster Operator* catalog subscription
+
+.Procedure
+
+. From the {product-title} web console, select `kubevirt-hyperconverged` from
+the *Projects* list.
+
+. Navigate to the *Catalog* -> *Operator Management* page.
+
+. Click the *Operator Subscriptions* tab.
+
+. Click the Options menu {kebab} in the row that contains the *hco-subscription*
+subscription. In the expanded menu, click *Remove Subscription*.
```

Comment:
@ousleyp  there is also a tick to remove Operator (see the screenshot attached). If no tick, it (namely all operator pods in Workloads -> Pods) won't be removed completely.

![Screenshot from 2019-06-21 13-58-44](https://user-images.githubusercontent.com/6319198/59921095-ee5c2980-942c-11e9-8958-26f45b1f49d4.png)